pool/doc.go

```diff
@@ -34,6 +39,19 @@ Execute NeoFS operation on the server:
 	res, err := p.PutContainer(context.Background(), prm)
 	// ...
 
+Execute NeoFS operation on the server, with disabled auto-session:
+
+	var prm pool.PrmContainerPut
+	prm.SetContainer(cnr)
+	// disable auto-session.
+	prm.DisableAutoSession()
+	// providing session manually.
+	prm.WithinSession(session)
+	// ...
+
+	res, err := p.PutContainer(context.Background(), prm)
```

Comment:
Examples are examples and not comments, #204.

Reply:
Moved to examples